### PR TITLE
cargo guidebook

### PR DESCRIPTION
**Pull request by Nails-n-Tape**
## About the PR
adds a guidebook entry for cargo

**Media**

cge1

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: added a guidebook entry for cargo

### CODE REVIEW

**Comment by metalgearsloth** [comment]
Please fix your CL.

**Comment by OttoMaticode** [comment]
> Please fix your CL.

fixed, the BTS work got a little messy.

**Comment by OttoMaticode** [comment]
also R.I.P. playable species guidebook that i accidentally obliterated during the push 😭

**Comment by OttoMaticode** [comment]
im just gonna leave the species guidebook untouched until this merges

File: Resources/ServerInfo/Guidebook/Cargo.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<Document>
+  # Cargo
+  The supply department, more commonly known simply as Cargo, is in charge of keeping the station stocked up on necessary resources, such as fuel and raw materials. Though, they are not restricted to simple goods, and can order a wide variety of different items from offstation. The point is, it's Cargo's job to get the rest of the station what they need.
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Cargo.xml:3]:
```suggestion
  The supply department, more commonly known simply as Cargo, is in charge of keeping the station stocked up on necessary resources, such as fuel and raw materials. Though, they are not restricted to simple goods, and can order a wide variety of different items from off-station. The point is, it's Cargo's job to get the rest of the station what they need.
```

File: Resources/ServerInfo/Guidebook/Cargo.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  ## Ordering
+  <Box>
+    <GuideEntityEmbed Entity="ComputerCargoOrders"/>
+  </Box>
+  Every transaction you make will begin at a [color=#a4885c]cargo request computer[/color], from which you can send orders for specific items. Making any order requires [color=#118C4F]spacebucks[/color], but we'll get to that later. For now, click the "request" button next to whatever you need, type an appropriate reason for why you're ordering it, and hit "approve" when it appears below the request list.
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Cargo.xml:9]:
```suggestion
  Every transaction you make will begin at a [color=#a4885c]cargo request computer[/color], from which you can make orders for specific products. Making any order requires [color=#118C4F]spacebucks[/color], the currency of the station. To make an order, click the "request" button next to whatever you need, type an appropriate reason for why you're ordering it, and hit "approve" when it appears below the request list.
```

File: Resources/ServerInfo/Guidebook/Cargo.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  ## Piloting
+  Alright! now that we've ordered an item, we have to get it to the station. Orders are shipped to the station via the [color=#a4885c]cargo shuttle[/color], the remotely piloted spacecraft that cargo uses to buy and sell goods. In order to move the shuttle, you must first find the [color=#a4885c]cargo shuttle computer[/color].
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Cargo.xml:12]:
```suggestion
  Orders are shipped to the station via the [color=#a4885c]cargo shuttle[/color], the remotely piloted spacecraft that cargo uses to buy and sell goods. In order to move the shuttle, you must first find the [color=#a4885c]cargo shuttle computer[/color].
```

File: Resources/ServerInfo/Guidebook/Cargo.xml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  <Box>
+    <GuideEntityEmbed Entity="ComputerCargoShuttle"/>
+  </Box>
+  After clicking the computer, you should see a radar-esque view of the shuttle from the top-down. Here are the steps for piloting the shuttle back and forth from the station, to the trading post, and back.
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Cargo.xml:16]:
```suggestion
  After clicking the computer, you should see a radar view of the shuttle. Here are the steps for piloting the shuttle back and forth:
```

File: Resources/ServerInfo/Guidebook/Cargo.xml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+  - First, disconnect any airlocks that are connected to the dock.
+  - Then, you actually get to pilot the shuttle. The controls are fairly simple, with W and S being forward and backward, A and D being turn left and right; and Spacebar being the brake.
+  - Move the shuttle a decent distance away from the station. After getting sufficiently far away, a FTL destination named something like "Trading Post Alpha" should light up on the computer. Hit this button, and the shuttle will start traveling to the trading post.
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Cargo.xml:20]:
```suggestion
  - Find the button that designates the trading post. Hit this button, and the shuttle will start traveling to it.
```
that feature was removed, fyi

File: Resources/ServerInfo/Guidebook/Cargo.xml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  - After some time, you should get the option to FTL the shuttle back to the station. After this, just pilot the shuttle back to the cargo dock, and it should have everything you ordered inside. Enjoy!
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Cargo.xml:21]:
```suggestion
  - After a short delay, the option to FTL back to the station will reappear. After this, just pilot the shuttle back to the cargo dock and retrieve the items you ordered.
```

File: Resources/ServerInfo/Guidebook/Cargo.xml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+  ## Selling
+  So, you want to order something, but don't have the money for it? You just need to sell something! Almost everything on the station has economic value, which an [color=#a4885c]appraisal tool[/color] can read out to you.
+  <Box>
+    <GuideEntityEmbed Entity="AppraisalTool"/>
+  </Box>
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Cargo.xml:27]:
```suggestion
    <GuideEntityEmbed Entity="AppraisalTool"/>
    <GuideEntityEmbed Entity="CargoPallet"/>
  </Box>
```
don't need to describe how it looks if you just include a picture

File: Resources/ServerInfo/Guidebook/Cargo.xml [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+  After clicking the computer, you should see a radar-esque view of the shuttle from the top-down. Here are the steps for piloting the shuttle back and forth from the station, to the trading post, and back.
+
+  - First, disconnect any airlocks that are connected to the dock.
+  - Then, you actually get to pilot the shuttle. The controls are fairly simple, with W and S being forward and backward, A and D being turn left and right; and Spacebar being the brake.
+  - Move the shuttle a decent distance away from the station. After getting sufficiently far away, a FTL destination named something like "Trading Post Alpha" should light up on the computer. Hit this button, and the shuttle will start traveling to the trading post.
+  - After some time, you should get the option to FTL the shuttle back to the station. After this, just pilot the shuttle back to the cargo dock, and it should have everything you ordered inside. Enjoy!
+
+  ## Selling
+  So, you want to order something, but don't have the money for it? You just need to sell something! Almost everything on the station has economic value, which an [color=#a4885c]appraisal tool[/color] can read out to you.
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Cargo.xml:24]:
```suggestion
  So now you know how to order something, but what if you don't have the money for it? You just need to sell something! Many items on the station have economic value, which can be read out using an [color=#a4885c]appraisal tool[/color].
```

File: Resources/ServerInfo/Guidebook/Cargo.xml [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+  <Box>
+    <GuideEntityEmbed Entity="AppraisalTool"/>
+  </Box>
+  After finding something worth selling, just heave it onto one of the shuttle's cargo pallets (they look like catwalks, and orders always spawn on them). The next time the shuttle arrives at the trading post, the item will be sold, and the money will be directly transfered back to the request computer.
```
Comment (EmoGarbage404) [inline Resources/ServerInfo/Guidebook/Cargo.xml:28]:
```suggestion
  After finding something worth selling, place it on one of the shuttle's cargo pallets. The next time the shuttle is sent to a trading post, the item will be sold and the money will be directly transferred back to the station's bank account.
```

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
wubwubwubwub

File: Resources/ServerInfo/Guidebook/Cargo.xml (hunk 8 above)
Comment (OttoMaticode) [reply Resources/ServerInfo/Guidebook/Cargo.xml:24]: Aight I believe what I did should just copy all of that

**Review by EmoGarbage404 — APPROVED** [review]
it's... it's... PERFECT

**Comment by Nails-n-Tape** [comment]
Grogchamp

**Comment by crazybrain23** [comment]
Bro used his master branch to make a pull request and it got merged.

**Pull request merged**